dashboard/modules/job/cli.py

```diff
@@ -270,18 +278,32 @@ def submit(
         runtime_env_json=runtime_env_json,
         working_dir=working_dir,
     )
-    job_id = client.submit_job(
-        entrypoint=list2cmdline(entrypoint),
-        submission_id=submission_id,
-        runtime_env=final_runtime_env,
-        metadata=metadata_json,
-        entrypoint_num_cpus=entrypoint_num_cpus,
-        entrypoint_num_gpus=entrypoint_num_gpus,
-        entrypoint_memory=entrypoint_memory,
-        entrypoint_resources=entrypoint_resources,
-    )
 
-    _log_big_success_msg(f"Job '{job_id}' submitted successfully")
+    if os.environ.get("RAY_JOB_SUBMISSION_ALLOW_EXISTS", "false") == "true":
+        allow_exists = True
+
+    try:
+        job_id = client.submit_job(
+            entrypoint=list2cmdline(entrypoint),
+            submission_id=submission_id,
+            runtime_env=final_runtime_env,
+            metadata=metadata_json,
+            entrypoint_num_cpus=entrypoint_num_cpus,
+            entrypoint_num_gpus=entrypoint_num_gpus,
+            entrypoint_memory=entrypoint_memory,
+            entrypoint_resources=entrypoint_resources,
+        )
+
+        _log_big_success_msg(f"Job '{job_id}' submitted successfully")
+    except Exception as e:
+        if (
+            allow_exists
+            and f"Job with submission_id {submission_id} already exists" in str(e)
```

Comment:
I am not sure whether the owner is okay with this or not. It's a bit hacky. Add some comments above.